go.mod

```diff
@@ -0,0 +1,175 @@
+	k8s.io/metrics => k8s.io/metrics v0.0.0-20191016113814-3b1a734dba6e
+	k8s.io/sample-apiserver => k8s.io/sample-apiserver v0.0.0-20191016112829-06bb3c9d77c9
+)
+
+// Fix tooling version
+replace (
+	// next line until pr https://github.com/ianlancetaylor/cgosymbolizer/pull/8 is merged
+	github.com/benesch/cgosymbolizer => github.com/truthbk/cgosymbolizer v0.0.0-20190731094043-91f7052d0d3a
+	github.com/fzipp/gocyclo => github.com/fzipp/gocyclo v0.0.0-20150627053110-6acd4345c835 // indirect
+	github.com/golangci/golangci-lint => github.com/golangci/golangci-lint v1.23.1
+	github.com/gordonklaus/ineffassign => github.com/gordonklaus/ineffassign v0.0.0-20200309095847-7953dde2c7bf // indirect
+	github.com/shuLhan/go-bindata => github.com/shuLhan/go-bindata v3.4.0+incompatible // indirect
+)
+
+// Internal deps fix version
+replace (
+	github.com/containerd/cgroups => github.com/containerd/cgroups v0.0.0-20200327175542-b44481373989
+	github.com/containerd/containerd => github.com/containerd/containerd v1.2.13
+	github.com/coreos/go-systemd => github.com/coreos/go-systemd v0.0.0-20180202092358-40e2722dffea
+	github.com/docker/distribution => github.com/docker/distribution v2.7.1-0.20190104202606-0ac367fd6bee+incompatible
+	github.com/florianl/go-conntrack => github.com/florianl/go-conntrack v0.1.1-0.20191002182014-06743d3a59db
+	github.com/iovisor/gobpf => github.com/DataDog/gobpf v0.0.0-20200131184214-6763fd92fd3f
+	github.com/mholt/archiver => github.com/mholt/archiver v2.0.1-0.20171012052341-26cf5bb32d07+incompatible
+	github.com/prometheus/client_golang => github.com/prometheus/client_golang v0.9.2
+	github.com/spf13/viper => github.com/DataDog/viper v1.7.1
+	github.com/ugorji/go => github.com/ugorji/go v1.1.7
+	google.golang.org/grpc => google.golang.org/grpc v1.23.0
+)
+
+require (
+	code.cloudfoundry.org/bbs v0.0.0-20200403215808-d7bc971db0db
+	code.cloudfoundry.org/cfhttp/v2 v2.0.0 // indirect
+	code.cloudfoundry.org/clock v1.0.0 // indirect
+	code.cloudfoundry.org/consuladapter v0.0.0-20200131002136-ac1daf48ba97 // indirect
+	code.cloudfoundry.org/diego-logging-client v0.0.0-20200130234554-60ef08820a45 // indirect
+	code.cloudfoundry.org/executor v0.0.0-20200218194701-024d0bdd52d4 // indirect
+	code.cloudfoundry.org/garden v0.0.0-20200224155059-061eda450ad9 // indirect
+	code.cloudfoundry.org/go-diodes v0.0.0-20190809170250-f77fb823c7ee // indirect
+	code.cloudfoundry.org/go-loggregator v7.4.0+incompatible // indirect
+	code.cloudfoundry.org/lager v2.0.0+incompatible
+	code.cloudfoundry.org/locket v0.0.0-20200131001124-67fd0a0fdf2d // indirect
+	code.cloudfoundry.org/rep v0.0.0-20200325195957-1404b978e31e // indirect
+	code.cloudfoundry.org/rfc5424 v0.0.0-20180905210152-236a6d29298a // indirect
+	code.cloudfoundry.org/tlsconfig v0.0.0-20200131000646-bbe0f8da39b3 // indirect
+	github.com/DataDog/agent-payload v0.0.0-20200407191957-2c57260621e5 // 4.30.0
+	github.com/DataDog/datadog-go v3.5.0+incompatible
```

Comment:
This got bumped from 2.2.0. Is it intended?

Reply:
upper is better?